bin/GameIndex.dbf

```diff
@@ -169,6 +169,15 @@ vuClampMode = 2 // Text in GT mode works
 //	
 //[/patches]
 ---------------------------------------------
+Serial = PCPX-96660
```

Comment:
Also move these two PCPX titles where the others are.

Reply:
This seems like too much work for contributors, is there any specific reason for needing this sorting order?

Reply:
They are in the good order, there is only 3 PCPX titles in the GameDB and they are after the PBPX titles so it should be fine :).

Reply:
Well apart from having thins tidy no. It will become quite a bit messy and harder to track so I prefer if we continue keeping it a bit cleaner. It's been done up until now.

Reply:
Anyway I will do this in the other open PR because of everybody tenis.

Reply:
GameDB is such a PITA to maintain which is why I stay away from it :stuck_out_tongue: